examples/fluids/qfunctions/newtonian.h

```diff
@@ -783,4 +785,58 @@ CEED_QFUNCTION(PressureOutflow_Jacobian_Prim)(void *ctx, CeedInt Q,
   return PressureOutflow_Jacobian(ctx, Q, in, out, StateFromY, StateFromY_fwd);
 }
 
+// *****************************************************************************
+// This QFunction implements the Harten Lax VanLeer (HLL) approximate Riemann
+// solver. Taking in two states.
+// *****************************************************************************
+CEED_QFUNCTION_HELPER StateConservative Harten_Lax_VanLeer_Flux(NewtonianIdealGasContext gas,
+    State left, State right, const CeedScalar normal[3]) {
+  // *INDENT-OFF*
+  // Inputs will be uleft and uright states
+
+  // Output will be the flux
+
+  // *INDENT-ON*
+  // Context
+
+  StateConservative flux_left = FluxInviscidDotNormal(gas, left, normal);
+  StateConservative flux_right = FluxInviscidDotNormal(gas, right, normal);
+
+  // Compute HLL flux
+  StateConservative Flux_HLL[3];
+  for (CeedInt i=0; i<3; i++) {
+  // compute speed
+
+  // change in states
+  CeedScalar deltas = right[i] - left[i];
+  CeedScalar s_left =  Dot3(left.Y.velocity, normal) - SoundSpeed(gas, left.Y.temperature);
+  CeedScalar s_right = Dot3(right.Y.velocity, normal) + SoundSpeed(gas, right.Y.temperature);
+  CeedScalar deltaF;
+  CeedScalar jumpspeed;
+  CeedScalar minspeed, maxspeed;
+  CeedScalar deltas1, deltas2;
+  if (fabs(deltas) <= 0.)
+    Flux_HLL[i] = flux_right;
```

Comment:
How about this to mirror the equation.
![image](https://user-images.githubusercontent.com/3303/198412648-4d697c7d-c89e-452a-a25a-0ac25d45f1a7.png)

```suggestion
  if (0 <= s_left)
    Flux_HLL = flux_left;
  else if (0 <= s_right)
    Flux_HLL = flux_right;
```